packages/demand-bloc-payment/src/constants.ts

```diff
@@ -4,6 +4,10 @@ export const defaultPaymentOptions = {
   paymentCardsEnabled: false,
 }
 
+export const defaultAdyenOptions = {
```

Comment:
Add a type for this variable? 
`const defaultAdyenOptions: AdyenOptions = { ...`